### PR TITLE
add dstu2 documentation for observation

### PR DESCRIPTION
**Pull request by dayouJ**

### CODE REVIEW

File: content/dstu2/observation.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+Search for labs, vitals, and alcohol/tobacco use Observations that meet supplied query parameters:
+
+    GET /Observation?:parameters
+
+### Parameters
+
+Name | Type | Description
+-----|------|--------------
+`subject`|`reference`| The subject that the observation is about.
+`patient`|`reference`| The subject that the observation is about (if patient).
```
Comment (dennispatterson) [inline content/dstu2/observation.md]: Nit: list params in the same order as the spec - http://hl7.org/fhir/2015May/observation.html#4.20.6

Comment (dayouJ) [inline content/dstu2/observation.md]: Changed.

**Comment by dayouJ** [comment]
Updated the pull request.

**Comment by dayouJ** [comment]
Updated the link.

File: content/dstu2/observation.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Search for labs, vitals, and alcohol/tobacco use Observations that meet supplied query parameters:
+
+    GET /Observation?:parameters
+
+### Parameters
+
+Name     | Type                                                           | Description
+---------|----------------------------------------------------------------|-------------------------------------------------------
+`patient`|[`reference`](http://hl7.org/fhir/2015May/search.html#reference)| The subject that the observation is about (if patient).
+`subject`|[`reference`](http://hl7.org/fhir/2015May/search.html#reference)| The subject that the observation is about.
```
Comment (naveenaluru) [inline content/dstu2/observation.md]: As discussed on diagnostic report PR, this should be 'subject:Patient'

**Comment by naveenaluru** [comment]
After addressing comments, :+1:

**Comment by dayouJ** [comment]
changed to subject:Patient

**Comment by dennispatterson** [comment]
:+1: after squash

**Pull request merged**